.pre-commit-config.yaml

```diff
@@ -16,14 +16,18 @@ repos:
     hooks:
     -   id: flake8
 -   repo: https://github.com/PyCQA/pylint
-    rev: v2.12.1
+    rev: v2.13.7
     hooks:
     -   id: pylint
         files: ^(telegram|examples)/.*\.py$
         args:
           - --rcfile=setup.cfg
           #  run pylint across multiple cpu cores to speed it up-
-          - --jobs=0  # See https://pylint.pycqa.org/en/latest/user_guide/run.html?#parallel-execution to know more
+          # https://pylint.pycqa.org/en/latest/user_guide/run.html?#parallel-execution to know more
+          - --jobs=0
+          # Warns about unused pylint ignores:
+          - --enable=useless-suppression
```

Comment:
if possible, please directly add this to `setup.cfg` so that it's also active when running pylint manually